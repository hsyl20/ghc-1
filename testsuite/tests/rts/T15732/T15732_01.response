+RTS -F3 -RTS
