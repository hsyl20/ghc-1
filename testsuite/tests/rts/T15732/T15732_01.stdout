oldGenFactor: 3.0
